app/lib/shared/handler_helpers.dart

```diff
@@ -19,21 +19,26 @@ import 'handlers.dart';
 import 'markdown.dart';
 import 'utils.dart' show fileAnIssueContent;
 
-Future runHandler(Logger logger, shelf.Handler handler,
-    {bool sanitize = false}) {
+Future<void> runHandler(
+  Logger logger,
+  shelf.Handler handler, {
+  bool sanitize = false,
+}) async {
   handler = _uriValidationRequestWrapper(handler);
   handler = _userAuthParsingWrapper(handler);
   if (sanitize) {
     handler = _sanitizeRequestWrapper(handler);
   }
   handler = _redirectToHttpsWrapper(handler);
   handler = _logRequestWrapper(logger, handler);
-  return runAppEngine((HttpRequest request) => _handleRequest(request, handler),
-      shared: true);
-}
-
-Future _handleRequest(HttpRequest request, shelf.Handler handler) =>
+  await runAppEngine((HttpRequest request) {
+    if (request.headers['x-appengine-cron'].contains('true') &&
```

Comment:
Wouldn't it be better to have a strict equals check?

Reply:
This is a list of headers... a header may have more than one value...

Reply:
Ah, yeah.